modules/javafx.controls/src/shims/java/javafx/scene/control/ControlShim.java

```diff
@@ -28,6 +28,19 @@
 
 public class ControlShim extends Control {
 
+    /**
+     * Installs the default skin for the given control.
+     *
+     * Note that this has no noticeable effect if the the control's
+     * skin already is set to the default skin (see skinProperty for
+     * implementations details).
+     *
+     * @param control the control the set the default skin on
```

Comment:
"the control the set the default skin on" --> "the control to set the default skin on"